src/NHibernate.Test/Insertordering/InsertOrderingFixture.cs

```diff
@@ -33,7 +34,7 @@ protected override string MappingsAssembly
 		protected override bool AppliesTo(Dialect.Dialect dialect)
 		{
 			// Custom batcher enforces sql server.
-			return dialect.SupportsSqlBatches;
+			return dialect is MsSql2000Dialect;
```

Comment:
`SupportsSqlBatches` is only about the `GO` statement handled by SQL Server Client tools. All its usages in tests were invalids.